src/or/channeltls.c

```diff
@@ -1705,7 +1721,7 @@ channel_tls_process_netinfo_cell(cell_t *cell, channel_tls_t *chan)
   /* Act on apparent skew. */
   /** Warn when we get a netinfo skew with at least this value. */
 #define NETINFO_NOTICE_SKEW 3600
-  if (labs(apparent_skew) > NETINFO_NOTICE_SKEW &&
+  if (time_abs(apparent_skew) &&
```

Comment:
Deleting "> NETINFO_NOTICE_SKEW" is not documented in the changes file: we'll now warn on any skew, not just skews over 1 hour.

Reply:
Ug, that's a bug.